test/priority-tests.js

```diff
@@ -91,4 +91,19 @@ test('compose override priorities', nest => {
       'staticProperties should override deepStaticProperties');
     assert.end();
   });
+
+  nest.test('...with deepStaticProperties', assert => {
+    const deepInstance = { e: 'deep' };
+    const stamp = compose({
+      deepStaticProperties: {
+        d: deepInstance
+      }
+    });
+    const actual = stamp.d;
+    const notExpected = stamp.compose.deepStaticProperties.d;
+
+    assert.notEqual(actual, notExpected,
```

Comment:
I don't see how this test would ever pass. The stamp has no `staticProperties` that would override the `deepStaticProperties`. `actual` and `notExpected` are the same here.